sinker/sinker.go

```
@@ -136,7 +136,20 @@ func (s *PostgresSinker) applyDatabaseChanges(dbChanges *pbddatabase.DatabaseCha
 			)
 		}
 
-		primaryKey := change.Pk
+		var primaryKey map[string]string
+		switch u := change.PrimaryKey.(type) {
+		case *pbddatabase.TableChange_Pk:
+			{
+				primaryKey, _ = s.loader.GetPrimaryKey(change.Table, u.Pk)
```

We must not ignore the error or make `GetPrimaryKey` infallible

---

I'll return the error right away and fail fast!